crates/superposition_types/src/lib.rs

```diff
@@ -222,7 +234,7 @@ impl RegexEnum {
     fn get_error_message(&self) -> String {
         match self {
             Self::DefaultConfigKey => ALPHANUMERIC_WITH_DOT_WORDS,
-            Self::DimensionName => ALPHANUMERIC_WITH_DOT_WORDS,
+            Self::DimensionName => ALPHANUMERIC_WITHOUT_DOT,
```

Comment:
This is wrong

Reply:
Can you please elaborate, what's wrong?

Reply:
If you will check [here](https://github.com/juspay/superposition/blob/fix%2Fdelimiter/crates%2Fsuperposition_types%2Fsrc%2Flib.rs#L249)
You can see for DimensionName, the regex is for with dot and you have changed the error message to without dot

Reply:
I meant to change the regex. My bad didn't confirm before pushing this change.